src/gtfs/read.rs

```diff
@@ -96,13 +107,41 @@ struct Stop {
     #[serde(rename = "stop_url")]
     url: Option<String>,
     #[serde(default)]
-    location_type: i32,
+    location_type: StopLocationType,
     parent_station: Option<String>,
     #[serde(rename = "stop_timezone")]
     timezone: Option<String>,
     #[serde(default)]
     wheelchair_boarding: Option<String>,
 }
+
+pub struct EquipmentList {
+    equipments: Vec<objects::Equipment>,
+    current_unaltered_id: u64,
+}
+
+impl Default for EquipmentList {
+    fn default() -> Self {
+        EquipmentList {
+            equipments: vec![],
+            current_unaltered_id: 1,
```

Comment:
you can use `equipments.len()` to generate the id instead of managing that by hand.